src/build.rs

```diff
@@ -49,17 +50,80 @@ fn rustc_minor_version() -> Option<u32> {
     otry!(pieces.next()).parse().ok()
 }
 
-/// Ensure that `rustup` has the `wasm32-unknown-unknown` target installed for
-/// current toolchain
-pub fn rustup_add_wasm_target(step: &Step) -> Result<(), Error> {
-    let msg = format!("{}Adding the Wasm target...", emoji::TARGET);
-    PBAR.step(step, &msg);
+/// Get rustc's sysroot as a PathBuf
+fn get_rustc_sysroot() -> Result<PathBuf, failure::Error> {
+    let command = Command::new("rustc")
+        .args(&["--print", "sysroot"])
+        .stdout(Stdio::piped())
```

Comment:
This can actually be removed since it's the default when calling `.output()`

Reply:
Oh right, thanks!